packages/browser-extension/source/localization/en_EN.ts

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
we should not have this file here

Reply:
this file is related to this project, en_EN.json is related to react app project